homeassistant/components/hue/sensor.py

```diff
@@ -113,25 +114,37 @@ def unit_of_measurement(self):
 
 SENSOR_CONFIG_MAP.update(
     {
-        TYPE_ZLL_LIGHTLEVEL: {
-            "platform": "sensor",
-            "name_format": LIGHT_LEVEL_NAME_FORMAT,
-            "class": HueLightLevel,
-        },
-        TYPE_ZLL_TEMPERATURE: {
-            "platform": "sensor",
-            "name_format": TEMPERATURE_NAME_FORMAT,
-            "class": HueTemperature,
-        },
-        TYPE_ZLL_SWITCH: {
-            "platform": "sensor",
-            "name_format": REMOTE_NAME_FORMAT,
-            "class": HueBattery,
-        },
-        TYPE_ZLL_ROTARY: {
-            "platform": "sensor",
-            "name_format": REMOTE_NAME_FORMAT,
-            "class": HueBattery,
-        },
+        TYPE_ZLL_LIGHTLEVEL: [
+            {
+                "platform": "sensor",
+                "name_format": LIGHT_LEVEL_NAME_FORMAT,
+                "class": HueLightLevel,
+            }
+        ],
+        TYPE_ZLL_TEMPERATURE: [
+            {
+                "platform": "sensor",
+                "name_format": TEMPERATURE_NAME_FORMAT,
+                "class": HueTemperature,
+            }
+        ],
+        TYPE_ZLL_SWITCH: [
+            {
+                "platform": "sensor",
+                "name_format": REMOTE_NAME_FORMAT,
+                "class": HueBattery,
+            }
+        ],
+        TYPE_ZLL_ROTARY: [
+            {
+                "platform": "sensor",
+                "name_format": REMOTE_NAME_FORMAT,
+                "class": HueBattery,
+            }
+        ],
     }
 )
+
+SENSOR_CONFIG_MAP[TYPE_ZLL_PRESENCE].extend(
+    [{"platform": "sensor", "name_format": REMOTE_NAME_FORMAT, "class": HueBattery}]
+)
```

Comment:
This relies on the fact that binary sensor is imported first. That is very frugal. Also use `append` instead of `extend` with a list with a single entry.
```suggestion
SENSOR_CONFIG_MAP.setdefault(TYPE_ZLL_PRESENCE, []).append(
    {"platform": "sensor", "name_format": REMOTE_NAME_FORMAT, "class": HueBattery}
)
```
Same has to be done for binary sensor.